app/models/invoice.rb

```diff
@@ -10,6 +10,18 @@ def self.find_from(customer_id)
     where(customer_id: customer_id)
   end
 
+  def status_view_format
```

Comment:
Misc thought/loose suggestion: This whole method can be reduced to `status.titleize`!

Reply:
this is great! I never heard of the .titleize method

Reply:
I made and pushed this change. TIL!